packages/ui-library/src/components/modal/index.tsx

```diff
@@ -182,14 +167,19 @@ const Modal: React.FC<ModalProps> = ({
                 variant={action.variant ?? 'contained'}
                 color={action.color ?? 'primary'}
                 size="large"
-                fullWidth
                 key={index}
+                sx={{ width: '100%' }}
               >
                 {action.label}
               </Button>
             )
           )}
-        </StyledDialogActions>
+          {showCloseButton && (
+            <Button onClick={onClose} variant="outlined" size="large" sx={{ width: '100%' }}>
```

Comment:
Doesn't native fullWidth work in thi case? Same for L171

Reply:
It adds a few more styles that we dont want